cfme/common/__init__.py

```diff
@@ -365,6 +365,41 @@ def step(self, *args, **kwargs):
         self.prerequisite_view.toolbar.policy.item_select('Manage Policies')
 
 
+class ComparableMixin:
+    """
+    Mixin for comparing entities. Should be added to Collection class.
+    Args:
+            provider: An InfraProvider instance
+            entities_list: A list of entities to compare
+    Usage:
+        If added to collection's class,
+
+        entity_coll = collection.all()[:2]
+        compare_view = collection.compare_entities_col(provider, entities_list=entity_coll)
+
+        will navigate to the correct view, select the items in entity_col (first two items returned
+        from all()), click on the compare dropdown item and verify the compare view is
+        displayed. It then returns the compare view.
+    """
+    DROPDOWN_TEXT = 'Compare Selected items'
+    NAV_STRING = 'All'
```

Comment:
I think `COMPARE_VIEW` and `COMPARE_APP_VIEW` should have a default value or at least be explicitly declared in the mixin. Could you also put them in the docblock?

Reply:
Added to docblock and added to mixin class. I used a string default stating to overwrite. I could add the classes that HostsCollection (or another entity class) uses, but I'd need to import the classes and it seems a bit wonky because each class usually uses a unique value for this.

Reply:
What about them defaulting to the generic class that you've defined below?

Reply:
Updated as suggested if I understood correctly.